src/stream/mod.rs

```diff
@@ -8,10 +8,13 @@ pub use transport::{StreamReadHalf, StreamTransport, StreamWriteHalf};
 
 #[cfg(feature = "unix-stream")]
 mod impl_unix_stream {
+	use std::future::Future;
+	use std::pin::Pin;
```

Comment:
These two lines are used by both the modules so can i not be moved above?

Reply:
Both modules are only conditionally enabled. Moving the import out of the modules will cause warnings about unused imports when both modules are disabled.